uberenv.py

```diff
@@ -365,7 +365,29 @@ def setup_paths_and_dirs(self):
 
         UberEnv.setup_paths_and_dirs(self)
 
-        self.ports = pjoin(self.uberenv_path, "vcpkg_ports","*")
+        # Find path to vcpkg ports
+        _errmsg = ""
+        if not self.opts["vcpkg_ports_path"] is None:
```

Comment:
Just below is used the much more human-readable `if foo is not None`. I suggest we use the same syntax here.

Reply:
did this and all other instances in script.